core/src/main/java/feign/Feign.java

```diff
@@ -289,7 +310,8 @@ public Feign build() {
       ParseHandlersByName handlersByName =
           new ParseHandlersByName(contract, options, encoder, decoder, queryMapEncoder,
               errorDecoder, synchronousMethodHandlerFactory);
-      return new ReflectiveFeign(handlersByName, invocationHandlerFactory, queryMapEncoder);
+      return new ReflectiveFeign(handlersByName, invocationHandlerFactory,
+              queryMapEncoder, methodHandlerCustomizers);
```

Comment:
this customizer basically acts just like one of `enrich` methods of `Capability` but I think it's too low level to be in there. Or is it?